lib/shared/drive-constraints.js

```diff
@@ -278,6 +278,23 @@ exports.isDriveSizeRecommended = (drive, image) => {
   return _.get(drive, [ 'size' ], UNKNOWN_SIZE) >= _.get(image, [ 'recommendedDriveSize' ], UNKNOWN_SIZE)
 }
 
+/**
+ * @summary 64GB
+ */
+const LARGE_DRIVE_SIZE = 64000000000
```

Comment:
I wonder if this should be base-2 units i.e. 68719476736 ?

Reply:
Storage devices usually have sizes in base-10, so base-2 wouldn't trigger on a 64GB SD-card, for example. Guess that depends on when we want that to happen.

Reply:
Yeah, I mentioned in flowdock that we probably _don't_ want this triggering on 64 GB SD cards, as I suspect they'll continue to become more prevalent as their price continues to drop. In my extensive experience, SD cards are always some-number-of-megabytes smaller than their stated capacity, so no 64GB card has a capacity of 65,536 MB.
For the record, the "64 GB" SD cards I have have exact byte-capacities of 62587404288, 62730010624, 62881005568, 63416827904, 64289243136. (even the biggest one has a capacity of 'only' 61,331 MB)

And I've never seen a 64GB hard drive, they always seem to be a "round" number of units: 80GB, 100GB, 2TB, etc.

Reply:
OTOH if we _do_ want this to trigger for 64 GB SD cards too, then it would probably make sense (for similar reasons as above) to set this to the base-2 32GB size, i.e. 34359738368 and this would then also catch 40 GB hard drives but no "32 GB" SD cards.